src/containers/pages/JurisdictionAssignment/JurisdictionTable/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -3,6 +3,7 @@
 exports[`src/containers/pages/jurisdictionView/jurisdictionTable selects and deselect nodes: should be now checked 1`] = `
 <input
   checked={true}
+  disabled={true}
```

Comment:
Could we add a test that shows this as true:

- For FI/Dynamic-FI
- For other plan types